### PR TITLE
#3 Provide AKS instructions

### PR DESCRIPTION
**Pull request by agrimmer**

### CODE REVIEW

File: content/docs/develop/installation/setup-keptn/index.md [hunk 1]
```diff
@@ -379,7 +401,7 @@ in the version of the latest release. Since v0.3 of keptn, the install command a
 
   ``` console
   git  clone --branch 0.2.2 https://github.com/keptn/installer
```
Comment (bacherfl) [inline content/docs/develop/installation/setup-keptn/index.md]: --branch 0.3.0

File: content/docs/develop/installation/setup-keptn/index.md [hunk 2]
```diff
@@ -167,6 +167,22 @@ Select one of the following options:
 </p>
 </details>
 
+<details><summary>Azure Kubernetes Service (AKS)</summary>
+<p>
+
+1. Install local tools
+  - [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+2. Create GKE cluster
```
Comment (bacherfl) [inline content/docs/develop/installation/setup-keptn/index.md]: AKS cluster

Comment (agrimmer) [reply content/docs/develop/installation/setup-keptn/index.md]: Thanks!

File: content/docs/develop/installation/setup-keptn/index.md (hunk 1 above)
Comment (agrimmer) [reply content/docs/develop/installation/setup-keptn/index.md]: Thanks!

**Pull request merged**